src/app/screens/Unlock/index.jsx

```diff
@@ -61,9 +65,18 @@ function Unlock() {
           primary
           disabled={password === ""}
         />
+
+        <div className="flex justify-center space-x-1 mt-5">
+          <span className="text-text-gray-bitcoin ">Need help? Contact </span>
+          <TextButton
```

Comment:
I don't see the need for a new TextButton component. This can be just a simple <a> link with a mailto: in the href.